rust/lang/security/audit/deserialize-untrusted.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+  - id: deserialize-untrusted-data
+    message: |
+      Data that is untrusted can not be trusted to be well-formed.
+      The application deserializes untrusted data without sufficiently verifying that the resulting data will be valid.
```

Comment:
I don't know the details with Rust. But in most other languages, verifying if the deserialized data is valid, is too late. Often remote code execution is possible **during** deserialization. Are you sure this is not also the case for Rust?

I don't see any resources listed in the references of this rule, so I cannot easily verify whether this is the case.

Reply:
https://www.reddit.com/r/rust/comments/8mjait/is_serialization_considered_a_security_risk/
https://github.com/serde-rs/serde/issues/1087

Reply:
https://serde.rs/